.rubocop.yml

```diff
@@ -1,4 +1,5 @@
 AllCops:
+  TargetRubyVersion: 2.4
```

Comment:
is there reason to still support out of support ruby 2.4 ? For 2.5 I know one SLE reason, but for 2.4 I am not sure.

Reply:
Right now supporting 2.4 is cheap.
As a general purpose library, I want to support even older versions if it is easy.